client/src/pages/Profile/ProfilePage.jsx

```diff
@@ -13,14 +13,25 @@ const ProfilePage = () => {
   useEffect(() => {
     const fetchData = async () => {
       try {
-        const response = await fetch("http://localhost:8080/api/get-workouts");
-        const responseData = await response.json();
-        setData(responseData);
-        // todo: until getUserById is implemented, we will hard code the user to be accountId=1
-        const responseProfile = responseData.filter(
-          (data) => data.accountId !== 1
-        );
-        setProfileData(responseProfile[0]);
+        const storedToken = sessionStorage.getItem("token");
```

Comment:
do you want to be using `localStorage` instead? `sessionStorage` expires after closing the page, requiring you to relog every time no?